app/components/AR.js

```diff
@@ -73,6 +73,7 @@ const HTML = `
 
 const BRIDGE_INJECT_SCRIPT = `
   var targetLocIdx = 0;
```

Comment:
`targetLocIndex` might be clearer, but that's just me. And/or do we want to standardize on `pins` over `locs`?